public/examples/google-pay.html

```diff
@@ -74,6 +49,31 @@
         }
       }
 
+      // Required in SCA Mandated Regions: Learn more at https://developer.squareup.com/docs/sca-overview
+      async function verifyBuyer(payments, token) {
+        const verificationDetails = {
+          amount: '1.00',
+          billingContact: {
+            addressLines: ['123 Main Street', 'Apartment 1'],
+            familyName: 'Doe',
+            givenName: 'John',
+            email: 'jondoe@gmail.com',
```

Comment:
```suggestion
            email: 'john.doe@square.example',
```

🙈 just in case that's someone's actual Gmail account

Reply:
Fair Point 🤣